stackslib/src/chainstate/stacks/boot/docs.rs

```diff
@@ -1,4 +1,5 @@
 use clarity::vm::docs::contracts::{produce_docs_refs, ContractSupportDocs};
+use clarity::vm::ClarityVersion;
```

Comment:
missing copyright comment block

Reply:
Added in 7a63f899e8945a770642517e96324330597dbe87.